modules/notes/controllers/deleteNote.js

```diff
@@ -0,0 +1,11 @@
+import { noteDelete } from '../../../db/providers/notes.js'
+
+const deleteController = async (req, res) => {
+  try {
+    const data = await noteDelete(req.params.id)
```

Comment:
data?

Reply:
Destructure please params as separate constants:
```suggestion
    const { id } = req.params;
    const deletedNote = await deleteNote(id);
```